### PR TITLE
Bug for single character colors in toHex()

### PR DESCRIPTION
**Pull request by pypmannetjies**
Hi,

I found an issue where, if a color was calculated and the result was a single character, the toHex() function would duplicate the character instead of prepending a 0. 

So for example: 
rgb(8,8,8) resolves to #888888 instead of #080808

I've edited the toHex() function to prepend a zero instead.

### CODE REVIEW

**Review by kcbleeker — APPROVED** [review]
This looks like a good idea!

**Comment by Olical** [comment]
Wow, this repo is a blast from the past! Past me was a fool (just like present me!) and didn't write any tests, so this feels like a change that will break things. This was from when I was first learning to write JavaScript in a way that I could give it to other people, feels so weird to read, like I didn't even write it!

Useless comments and imperative loops everywhere. :scream: Also, single-quotes are used everywhere else. As long as you're confident that my original code was just completely broken and this fixes it, I'm happy to merge.

File: color.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ var color = {
 			
 			// Make sure it is always the right length
 			if(color[i].length === 1) {
-				color[i] = color[i] + color[i];
+				color[i] = "0" + color[i];
```
Comment (Olical) [inline color.js]: Swap these to single quotes and I think this is fine. I _think_.

**Review by Olical — CHANGES_REQUESTED** [review]
Just that one tiny quote change.

File: color.js (hunk 1 above)
Comment (pypmannetjies) [reply color.js]: Ahaha yeah I saw you wrote this 6 years ago! I'll submit another pull request later today with the min file as well.

Comment (Olical) [reply color.js]: Feel free to modify this PR :) you should be able to add commits to it. Gonna write a post about what it feels like when some of your earliest stuff pops up out of nowhere :D I'll probably use this as a prompt to clean this project up, might as well.

Thanks!

**Comment by pypmannetjies** [comment]
Should be good now

**Review by Olical — APPROVED** [review]
Nice work. I'll comment here when I've finished my little cleanup post too :)

**Comment by pypmannetjies** [comment]
Thanks!

**Pull request merged**